trailblazer/clients/slurm_api_client/error_handler.py

```diff
@@ -0,0 +1,25 @@
+from functools import wraps
+import logging
+
+from pydantic import ValidationError
+import requests
+
+from trailblazer.exc import ResponseDeserializationError, SlurmAPIClientError
+
+LOG = logging.getLogger(__name__)
+
+
+def handle_errors(func):
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except requests.exceptions.HTTPError as e:
+            LOG.error(f"Error getting job: {e.response.content}")
```

Comment:
This logging is a bit misguiding since it would be the logging shown when a "cancel job" request fails. Previous logging in that instance was:
"LOG.error(f"Error cancelling job {job_id}: {e.response.content}")"